### ISSUE TITLE
Names of suites and resources should not be normalized

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#122]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86) by @yanne on 27 Jan 2009*

Robot Framework style of normalization of suite and resource names should
be dropped in favor of simpler approach where only underscores are
substituted with spaces, but capitalization is left as it is on the file names.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c1) by @yanne on 27 Jan 2009*


Implementation in <a href="https://code.google.com/p/robotframework-ride/source/detail?r=244">r244</a>, waiting for customer approval

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c2) by @pekkaklarck on 27 Jan 2009*


There's a slight problem here that RF itself handles names differently. First of all
it's illogical that names appear differently in RIDE and in RF outputs. The bigger
problem is that, if I understood the change in <a href="https://code.google.com/p/robotframework-ride/source/detail?r=244">r244</a> correctly, suite names that are
got from RF modules inside the parsed test suite will anyway be handled as earlier.
Doesn't that also mean that if RIDE reloads date it will be displayed differently?

Another issue is that printable_name_from_path in RF also converts 'MyName' to 'My
Name' which the RIDE implementation doesn't do.

Let's talk about this on Wednesday. If my concerns above are valid, it might be
better to change this in RF itself first.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c3) by @yanne on 9 Feb 2009*


Decided to hold this until a consistent implementation available through RF.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c5) by @pekkaklarck on 22 Sep 2009*


This can be done only after RF 2.2 when this issue is implemented:
<a href="https://code.google.com/p/robotframework/issues/detail?id=377" rel="nofollow">https://code.google.com/p/robotframework/issues/detail?id=377</a>

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c6) by @pekkaklarck on 22 Sep 2009*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=205" title="RIDE shouldn't camelcase my test suite names"> Issue 205 </a> has been merged into this issue.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=86#c7) by @jussimalinen on 3 Sep 2010*


Done already in earlier 0.25 release.
